pipenv/core.py

```diff
@@ -1344,7 +1345,8 @@ def pip_install(
             index_source = index_source.copy()
         except SourceNotFound:
             src_name = project.src_name_from_url(index)
-            verify_ssl = True if index not in trusted_hosts else False
+            index_url = parse_url(index)
+            verify_ssl = True if index_url.host not in trusted_hosts else False
```

Comment:
I believe that the following form is enough:
```python
verify_ssl = index_url.host not in trusted_hosts
````

Reply:
Indeed it is! thanks for the feedback :)

Reply:
bigger fish to fry on this one -- VCS installs are throwing permissions errors :(